Assets/Scripts/Core/Controllers/GameController.cs

```diff
@@ -64,6 +64,14 @@ public class GameController : MonoBehaviour {
 			} else {
 				Debug.Log(SceneManager.GetActiveScene().name + " has no level to load!");
 			}
+			if (SceneManager.GetActiveScene().name == "Final Level")
+			{
+				tilemap.Load("finallevel_V1.2");
+			}
+			else
```

Comment:
This `else` case can be removed in favor of the one above, if we chain everything with `else if`.